how-tos/manually-generate-partials.md

```diff
@@ -0,0 +1,104 @@
+* all locales are being generated, possibly only some of the platforms (which are win32 and win64 in this tutorial)
+
+# How? 
+
+## 1. build/tools changes
+
+```sh
+export FROM_VERSION='47.0.2'  # Change this
+export TO_VERSION='53.0'  # Change this
+export TO_BUILD='6'   # Change this
+export TO_VERSION_UNDERSCORE = ${TO_VERSION//./_} # Example: 53_0
+
+wget
+"https://hg.mozilla.org/releases/mozilla-release/raw-file/FIREFOX_${TO_VERSION_UNDERSCORE}_RELEASE/browser/locales/shipped-locales"                                             
+PERL5LIB=tools/lib/perl perl tools/release/patcher-config-bump.pl \
+  -c tools/release/patcher-configs/mozRelease-branch-patcher2.cfg \
+  -p firefox -r Firefox -v "$TO_VERSION" -a "$TO_VERSION" -o "$FROM_VERSION" -b "$TO_BUILD" \
+  -f archive.mozilla.org -d download.mozilla.org -l shipped-locales --partial-version "$FROM_VERSION" \
+  --platform linux --platform linux64 --platform macosx64 --platform win32 --platform win64
+
+python tools/scripts/build-promotion/create-update-verify-config.py \
+  --config tools/release/patcher-configs/mozRelease-branch-patcher2.cfg \
+  --platform win32 --update-verify-channel release-localtest \
+  --output tools/release/updates/release-firefox-win32.cfg \
+  --archive-prefix https://archive.mozilla.org/pub \
+  --previous-archive-prefix https://archive.mozilla.org/pub \
+  --product firefox --balrog-url https://aus5.mozilla.org --build-number "$TO_BUILD"
+  
+python tools/scripts/build-promotion/create-update-verify-config.py \
+  --config tools/release/patcher-configs/mozRelease-branch-patcher2.cfg \
+  --platform win64 --update-verify-channel release-localtest \
+  --output tools/release/updates/release-firefox-win64.cfg \
+  --archive-prefix https://archive.mozilla.org/pub \
+  --previous-archive-prefix https://archive.mozilla.org/pub \
+  --product firefox --balrog-url https://aus5.mozilla.org --build-number "$TO_BUILD"
+
+cd tools/
+
+# Edit release/updates/release-firefox-win32.cfg and release/updates/release-firefox-win64.cfg to only contain the line about $FROM_VERSION
+
+hg commit
+hg push -r . review   # Ask for a review
+
+# Once review is passed, retag the following:
+hg tag -f -r . "FIREFOX_${TO_VERSION_UNDERSCORE}_RELEASE_RUNTIME"
+hg tag -f -r .^ "FIREFOX_${TO_VERSION_UNDERSCORE}_BUILD${TO_BUILD}_RUNTIME"
+
+hg push -r .^^  # Actual changes
+hg push -r .^   # First tag
+hg push -r .    # Second tag
+```
+
+## 2. Generate taskgraph
+
+```sh 
+ssh buildbot-master85.bb.releng.scl3.mozilla.com
+sudo su - ctlbld
+mkdir bug-xxxxxx && cd bug-xxxxxx
+curl --location https://bugzilla.mozilla.org/attachment.cgi?id=8818876 > partials.tar.gz
+tar xvf partials.tar.gz
+cd partials
+cd partials-47.0.2-50.1.0/
+
+# Modify runme.py to point to $TO_VERSION
+# Fill the creds in config.yml
+source /builds/releaserunner/bin/activate
+python runme.py
+
+# If the graph is printed out, uncomment the last line of runme.py
+python runme.py
+```
+
+## 3. Update balrog blobs and bouncers
```

Comment:
Can you move this make it step 2, so we generate the graph only after the blobs are ready?